app/Http/Controllers/UserController.php

```diff
@@ -284,4 +290,42 @@ public function deleteData()
         return response()->json($results);
     }
 
+    public function SortPagination()
```

Comment:
Should unify the function name convention

Reply:
fixed